lib/samples/query_related_features/query_related_features.dart

```diff
@@ -0,0 +1,287 @@
+import 'dart:math';
+
+import 'package:arcgis_maps_sdk/arcgis_maps.dart';
+import 'package:flutter/material.dart';
+
+import '../../utils/sample_state_support.dart';
+
+class QueryRelatedFeatures extends StatefulWidget {
+  const QueryRelatedFeatures({super.key});
+
+  @override
+  State<QueryRelatedFeatures> createState() => _QueryRelatedFeaturesState();
+}
+
+class _QueryRelatedFeaturesState extends State<QueryRelatedFeatures>
+    with SampleStateSupport {
+  // Create a controller for the map view.
+  final _mapViewController = ArcGISMapView.createController();
+  // A flag for when the map view is ready and controls can be used.
+  var _ready = false;
+  // A flag for when the settings bottom sheet is visible.
+  var _layerDataVisible = false;
+  // A flag for when the features are loading.
+  var _loadingFeatures = false;
+  // Feature layer for the Alaska National Parks.
+  late final FeatureLayer _alaskaNationalParksLayer;
+  // Feature table for the Alaska National Parks features.
+  late final ServiceFeatureTable _alaskaNationalParksFeaturesTable;
```

Comment:
Does this need to be a member variable? it looks like it is only used in `onMapViewReady`

Reply:
Yes it looks like I can just declare it locally in the method along with other tables!